interfaces/devdoc/socket_transport_requirements.md

```diff
@@ -171,4 +178,12 @@ MOCKABLE_FUNCTION(, SOCKET_HANDLE, socket_transport_get_underlying_socket, SOCKE
 MOCKABLE_FUNCTION(, bool, socket_transport_is_valid_socket, SOCKET_TRANSPORT_HANDLE, socket_transport_handle);
 ```
 
-`socket_transport_check_valid_handle` checks that the internal socket is valid.
\ No newline at end of file
+`socket_transport_check_valid_handle` checks that the internal socket is valid.
+
+### socket_transport_get_local_address
+
+```c
+MOCKABLE_FUNCTION(, int, socket_transport_get_local_address, SOCKET_TRANSPORT_HANDLE, socket_transport, char, hostname[MAX_GET_HOST_NAME_LEN], LOCAL_ADDRESS**, local_address_list, uint32_t*, address_count);
+```
+
+`socket_transport_get_local_address` shall return the hostname and IP address of the machine.
```

Comment:
should it be plural? Does it return more than one address if there are multiple interface cards?

Reply:
good catch.